backend/controllers/default.py

```diff
@@ -11,3 +14,15 @@ async def root(request: Request):
         "success": True,
         "message": "Iceburg is running!",
     }
+
+
+@router.get("/user")
+async def get_rd_user():
+    api_key = settings_manager.get("realdebrid")["api_key"]
```

Comment:
this should be in realdebrid.py

Reply:
Did we want to keep all endpoints in controllers dir?

Reply:
> Did we want to keep all endpoints in controllers dir?

Yes, consistency will help in finding stuff 😄